vectorizers/_vectorizers.py

```diff
@@ -422,16 +422,16 @@ def preprocess_tree_sequences(
             result_sequence.append((result_matrix, label_sequence))
     else:
         result_sequence = []
-        if MASK_STRING in token_dictionary:
-            _ = token_dictionary.pop(MASK_STRING)
+        if masking in token_dictionary:
+            _ = token_dictionary.pop(masking)
```

Comment:
```suggestion
            del token_dictionary[masking]
```
is probably more traditional given that you don't care about the return value from pop.